docs/_menu_template.html

```diff
@@ -0,0 +1,16 @@
+            <div id="{{fsdocs-menu-header-id}}-wrapper" class="p-2 list-wrapper menu-open">
+            <li class="docs-menu">
+                <button class="btn text-white d-flex p-1 justify-content-between" data-bs-toggle="collapse"
+                        data-bs-target="#{{fsdocs-menu-header-id}}" aria-expanded="true">
+                        {{fsdocs-menu-header-content}}
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" stroke="currentColor" stroke-width="2" fill="currentColor" class="bi bi-chevron-down mx-1" viewBox="0 0 16 16">
```

Comment:
I'd go for an `<i>` tag here instead, something like:
```html
<i class="bi bi-chevron-down mx-2" style="-webkit-text-stroke: 1px;"></i>
```

Reply:
sure, do note that I added a vector because it's font counter part didn't allow me to resize it the way I liked. I'll look into it tho.